src/neo/IO/MemoryReader.cs

```diff
@@ -198,6 +198,9 @@ public string ReadFixedString(int length)
             int i = pos;
             while (i < end && span[i] != 0) i++;
             ReadOnlySpan<byte> data = span[pos..i];
+            for (; i < end; i++)
+                if (span[i] != 0)
+                    throw new FormatException();
             pos = end;
             return Utility.StrictUTF8.GetString(data);
```

Comment:
Maybe it's faster use whole the data (without checking 0) in `GetString` and after it, call `TrimEnd()`.

Reply:
If it is not followed by all 0s, it does not meet the format.

Reply:
I think that MySql only trim the last `\0'